### ISSUE TITLE
metrics about region captioning

### ISSUE THREAD

**Issue by Hoteryoung** [mbzuai-oryx/GeoChat#36]
I evaluated the VQA and scene cls tasks on the model fine-tuned using GeoChatInstruct,  and the results are pretty close to the metrics reported in the paper, however, the region captioning result is a bit far from the paper. 
The official evaluation result:
![image](https://github.com/mbzuai-oryx/GeoChat/assets/32992355/9080e252-3833-4dc8-bf8b-f06128ef1359)
My result:
![image](https://github.com/mbzuai-oryx/GeoChat/assets/32992355/579acccb-34d9-4156-9061-a2500e4cd847)
Note that:
1. I finetuned the model only the first stage, which means I finetuned the LLaVA-v1.5-7b using GeoChatInstruct for only one epoch, and I did not further fine-tune the model using only referring and grounding samples since the lack of details in the paper about the stage 2 fientune.
2. I used the **evaluate** package of HuggingFace.
I wonder whether I did something wrong or the metric gap is caused by the stage2 finetune?

**Comment by Davidup1** [comment]
@Hoteryoung I also met this problem and the metric fell even lower after stage2 finetune😂

**Comment by Oreouo** [comment]
Could you show me the code for calculating the metrics of your VQA?

**Comment by Oreouo** [comment]
Could you show me the code for calculating the metrics of your referring?
